<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICHCSettingsFrame</class>
 <widget class="QFrame" name="ICHCSettingsFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="ICPageTab" name="axisX1ToolButton" native="true">
       <property name="minimumSize">
        <size>
         <width>67</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>67</width>
         <height>34</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ICPageTab" name="axisY1ToolButton" native="true">
       <property name="minimumSize">
        <size>
         <width>67</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>67</width>
         <height>34</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ICPageTab" name="axisZToolButton" native="true">
       <property name="minimumSize">
        <size>
         <width>67</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>67</width>
         <height>34</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ICPageTab" name="axisX2ToolButton" native="true">
       <property name="minimumSize">
        <size>
         <width>67</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>67</width>
         <height>34</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ICPageTab" name="axisY2ToolButton" native="true">
       <property name="minimumSize">
        <size>
         <width>67</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>67</width>
         <height>34</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ICPageTab" name="axisAToolButton" native="true">
       <property name="minimumSize">
        <size>
         <width>67</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>67</width>
         <height>34</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ICPageTab" name="axisBToolButton" native="true">
       <property name="minimumSize">
        <size>
         <width>67</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>67</width>
         <height>34</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ICPageTab" name="axisCToolButton" native="true">
       <property name="minimumSize">
        <size>
         <width>67</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>67</width>
         <height>34</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>476</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="settingStackedFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>400</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICPageTab</class>
   <extends>QWidget</extends>
   <header>icpagetab.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
